app/src/pages/[resourceType]/[id].tsx

```diff
@@ -61,6 +65,15 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
               </Prism>
             </Tabs.Panel>
           )}
+          {decodedELM != null && (
+            <Tabs.Panel value="elm" pt="xs">
+              {/* eslint-disable  @typescript-eslint/no-explicit-any */}
+              <Prism language={'elm' as any} colorScheme="light">
```

Comment:
```suggestion
              <Prism language="json" colorScheme="light">
```
The ELM is JSON and highlighting is supported in Prism so you can just use "json" for the language. Similar to how it is used for the resource JSON just above. Only difference here is that we already have the JSON as a string.